### PR TITLE
Added example 3

### PR DESCRIPTION
**Pull request by Skatterbrainz**
Added example 3 to show multiple permissions, and appended description for example 1.

### CODE REVIEW

**Comment by e0i** [comment]
@Skatterbrainz Thank you for your contributions. The PR is being reviewed.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/smbshare/New-SmbShare.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Name                          ScopeName                     Path
 VMSFiles                      Contoso-SO                    C:\ClusterStorage\Volume1\...
 ```
 
-This command creates an SMB share.
+This command creates an SMB share named "VMSFiles" and grants Full Access permissions to "Contoso\Administrator", and "Contoso\Contoso-HV1$"
```
Comment (JohanFreelancer9) [inline docset/windows/smbshare/New-SmbShare.md]:
```suggestion
This command creates an SMB share named "VMSFiles" and grants Full Access permissions to "Contoso\Administrator" and "Contoso\Contoso-HV1$".
```

File: docset/windows/smbshare/New-SmbShare.md [hunk 2]
```diff
@@ -61,6 +61,18 @@ Data                          Contoso-FS                    J:\Data
...
+### Example 3: Create an SMB share with Multiple Permissions
+```
+PS C:\>New-SmbShare -Name "VMSFiles" -Path "C:\ClusterStorage\Volume1\VMFiles" -ChangeAccess "Users" -FullAccess "Administrators"
+
+Name                          ScopeName                     Path                          Description 
+----                          ---------                     ----                          ----------- 
+VMSFiles                      Contoso-SO                    C:\ClusterStorage\Volume1\...
+```
+
+This command creates an SMB share named "VMSFiles" and grants Change permissions to the local "Users" group, and Full Access permissions to the local "Administrators" group.
```
Comment (JohanFreelancer9) [inline docset/windows/smbshare/New-SmbShare.md]:
```suggestion
This command creates an SMB share named "VMSFiles", grants Change permissions to the local "Users" group, and Full Access permissions to the local "Administrators" group.
```

**Comment by get-itips** [comment]
I am ok once the copy review suggestions are accepted

**Comment by e0i** [comment]
@Skatterbrainz We would like to proceed with merge of this PR once the reviewer's suggestions have been applied in the PR. Thanks.

**Comment by e0i** [comment]
@Skatterbrainz We would like to proceed with merge of this PR once the reviewer's suggestions have been applied in the PR. Thanks.

**Comment by e0i** [comment]
@Skatterbrainz  Closing as there has been no follow-up with the reviewers' revision suggestions. Please re-open the PR to proceed with the review. Thank you for your patience with the process and your contributions.

**Comment by Skatterbrainz** [comment]
Sorry for the delayed response.  I was pulled into other matters and didn't get a chance to follow-up on this.

**Comment by e0i** [comment]
@Skatterbrainz 

> Sorry for the delayed response. I was pulled into other matters and didn't get a chance to follow-up on this.

Apologies as well. It seems that the PR has had gone off my radar because the PR had been closed and re-opened. Proceeding to merge. Any of your further contribtuions are appreciated. Thank you.

**Pull request merged**